### PR TITLE
Turned style guide into book

### PR DESCRIPTION
**Pull request by dariavladykina**
With @tomschr 's kind help, the style guide was turned into a book (structurally) and now enjoys the 3-column layout with subsections (which we have a lot) well seen on the right.

### CODE REVIEW

File: xml/docu_styleguide.content.xml
Comment (tomschr) [inline xml/docu_styleguide.content.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.language.xml
Comment (tomschr) [inline xml/docu_styleguide.language.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.manage.xml
Comment (tomschr) [inline xml/docu_styleguide.manage.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.name.xml
Comment (tomschr) [inline xml/docu_styleguide.name.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.outline.xml
Comment (tomschr) [inline xml/docu_styleguide.outline.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.smartdocs.xml
Comment (tomschr) [inline xml/docu_styleguide.smartdocs.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.structure.xml
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.taglist.xml
Comment (tomschr) [inline xml/docu_styleguide.taglist.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.webwriting.xml
Comment (tomschr) [inline xml/docu_styleguide.webwriting.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

File: xml/docu_styleguide.xmlformat.xml
Comment (tomschr) [inline xml/docu_styleguide.xmlformat.xml:1]: Also change the `sect1` in the DOCTYPE to `chapter`

**Review by tomschr — CHANGES_REQUESTED** [review]
@dariavladykina Thanks Daria, this looks great! :+1: There is only a minor issue in the DOCTYPE, see below.

**Review by janajaeger — APPROVED** [review]
Yayyy, finally! Thanks to the both of you!

**Review by jfaltenbacher — APPROVED** [review]
that is great. Looking forward to seeing the new book :)

**Review by PadraigDillon1 — APPROVED** [review]

**Comment by tomschr** [comment]
@dariavladykina I found two files which the wrong DOCTYPE. I fixed it in commit 05e97a4. Other than that, it looks great! :+1:

**Review by tomschr — APPROVED** [review]

**Pull request merged**